README.md

```diff
@@ -1,4 +1,7 @@
 # Module1
+-----------
+for REVIEW
+-----------
```

Comment:
Maybe put the labelled link to the live version above the description. It's a little buried here. :-)

Reply:
sure